src/main/java/org/jenkins/tools/test/model/PluginCompatTesterConfig.java

```diff
@@ -219,108 +209,9 @@ public Map<String, String> retrieveMavenProperties() throws IOException {
             }
         }
 
-        // Read other explicit CLI arguments
-
-        // Override JDK if passed explicitly
-        if (testJDKHome != null) {
-            if (!testJDKHome.exists() || !testJDKHome.isDirectory()) {
-                throw new IOException("Wrong Test JDK Home passed as a parameter: " + testJDKHome);
-            }
-
-            if (res.containsKey("jvm")) {
-                LOGGER.log(
-                        Level.WARNING,
-                        "Maven properties already contain the 'jvm' argument; "
-                                + "overriding the previous test JDK home value '"
-                                + res.get("jvm")
-                                + "' by the explicit argument: "
-                                + testJDKHome);
-            } else {
-                LOGGER.log(Level.INFO, "Using custom test JDK home: {0}", testJDKHome);
-            }
-            final String javaCmdAbsolutePath = getTestJavaCommandPath();
-            res.put("jvm", javaCmdAbsolutePath);
-        }
-
         return res;
     }
 
-    @CheckForNull
-    private String getTestJavaCommandPath() {
-        if (testJDKHome == null) {
-            return null;
-        }
-        return new File(testJDKHome, "bin/java").getAbsolutePath();
-    }
-
-    /**
-     * Gets the Java version used for testing, using the binary path to the <code>java</code>
-     * command.
-     *
-     * @return a string identifying the jvm in use
-     */
-    public String getTestJavaVersion() throws IOException {
```

Comment:
Was only ever used to populate a value in an XML report that nobody ever looked at.